cmd/build.go

```diff
@@ -17,6 +17,7 @@ func newLocalExecuteCommand(config *settings.Config) *cobra.Command {
 
 	local.AddFlagsForDocumentation(buildCommand.Flags())
 	buildCommand.Flags().StringP("org-slug", "o", "", "organization slug (for example: github/example-org), used when a config depends on private orbs belonging to that org")
+	buildCommand.Flags().String("org-id", "", "organization id, used when a config depends on private orbs belonging to that org")
```

Comment:
nitpick here but I'd argue keeping the wording the same (or extract to a variable even) for the three `<command>.Flags().String("org-id"...`

Reply:
Could you give a full example? The wording is the same as far as I can see...